Core/Core/Login/LoginFindSchoolViewController.swift

```diff
@@ -18,17 +18,21 @@
 
 import UIKit
 
-class LoginFindSchoolViewController: UIViewController, LoginFindSchoolViewProtocol {
-    @IBOutlet weak var keyboardSpace: NSLayoutConstraint?
-    @IBOutlet weak var loadingView: UIActivityIndicatorView?
-    @IBOutlet weak var promptLabel: DynamicLabel?
-    @IBOutlet weak var resultsTableView: UITableView?
-    @IBOutlet weak var searchField: UITextField?
+class LoginFindSchoolViewController: UIViewController {
+    @IBOutlet weak var keyboardSpace: NSLayoutConstraint!
+    @IBOutlet weak var loadingView: UIActivityIndicatorView!
+    @IBOutlet weak var promptLabel: UILabel!
```

Comment:
is this in the right font now?

Reply:
It still is a `DynamicLabel` in the storyboard with the font set up there. The controller doesn't need to care though. The extras added by `DynamicLabel` are only used by storyboards.

Reply:
cool cool cool